web-components/src/components/chip/Chip.ts

```diff
@@ -211,9 +211,7 @@ export class Chip extends LitElement {
       <md-tooltip ?disabled=${!this.textOverflow} message="${this.value}">
         <span
           tabindex="0"
-          class="md-chip${classMap(classNamesInfo)}"
-          aria-selected=${this.selected}
-          aria-readonly=${this.readonly}
```

Comment:
@futurethang  please review. what does removal of this aria-label achieves ?

Reply:
@yananym @helen-v These attributes should remain, I cannot se any reason to remove them. There was a useful change on line 214 though, adding a space between the `md-chip` class and mapped class. I see it in my latest pull of this branch but not here . .  I might need to just reload this page

Reply:
We have critical error if we have those attributes with role:

`Critical
Ensures ARIA attributes are allowed for an element's role
ARIA attributes are not allowed: aria-selected="false", aria-readonly="false"`

Resolved: returned attributes, but set role to the upper level